scripts/version.js

```diff
@@ -3,39 +3,17 @@ const path = require("path");
 const { execSync } = require("child_process");
```

Comment:
If I'm understanding correctly, `changesets` will do most of the versioning and publishing for us going forward.  Are there specific use cases we have for our own `version.js` and `publish.js` scripts?

Reply:
Echoing from Discord, but we'll keep these around for any one-off manually tagged releases (like experimental and nightly)